crates/bevy_render/src/render_graph/graph.rs

```diff
@@ -101,6 +103,70 @@ impl RenderGraph {
         id
     }
 
+    /// Removes the `node` with the `name` from the graph.
+    /// If the name is does not exist, nothing happens.
+    pub fn remove_node(&mut self, name: impl Into<Cow<'static, str>>) {
+        let name = name.into();
+        if let Some(id) = self.node_names.remove(&name) {
+            if let Some(node_state) = self.nodes.remove(&id) {
+                // Remove all edges from other nodes to this one. Note that as we're removing this
+                // node, we don't need to remove its input edges
+                for input_edge in node_state.edges.input_edges.iter() {
+                    match input_edge {
+                        Edge::SlotEdge {
+                            input_node: _,
+                            input_index: _,
+                            output_node,
+                            output_index: _,
+                        } => {
+                            if let Ok(output_node) = self.get_node_state_mut(*output_node) {
+                                output_node
+                                    .edges
+                                    .remove_output_edge(input_edge.clone())
+                                    .ok();
```

Comment:
The `.ok()` here doesn't do anything, is the intention of it to silence the "unused output" warning or is it a leftover from a previous version?

Reply:
maybe this method could return a `Result<(), RenderGraphError>` like the other methods

Reply:
Done.